cli_helpers/utils.py

```diff
@@ -34,7 +34,7 @@ def truncate_string(value, max_width=None, skip_multiline_string=True):
     if skip_multiline_string and isinstance(value, text_type) and '\n' in value:
         return value
     elif isinstance(value, text_type) and max_width is not None and len(value) > max_width:
-        return value[:max_width]
+        return value[:max_width-3] + "..."
```

Comment:
Should we add the `...` in the next line? 

The reason I ask is that we currently truncate strings that are bigger than 500 characters. Which is long enough that people are not going to do the horizontal scroll to see it.

Reply:
That might look better with pager that does not wrap long lines. But it will look odd with pager that does. I'm not sure of a good solution here, so I chose to be "less invasive".

Reply:
Good point.